view/app-jarvis.js

```diff
@@ -58,6 +58,136 @@ app.controller('MainController', function($scope,$location,$rootScope,$http) {
 		}
 	});
 
+	$scope.getResponse = function(messageObj) {
+		const	data = 'username='+USER+'&message='+messageObj.message;
```

Comment:
Please remove the trailing space between `const` and `data`. Also, kindly interpolate all the strings in the file to `${}` than using the plus symbol. That would be nice for the entire app.